client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -89,22 +99,87 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 					mainText={ __( 'Transfer this domain to any site you are an administrator on' ) }
 				/>
 			</Card>
+		);
+	};
+
+	const requestTransferCode = () => {
+		requestDomainTransferCodeOnly( selectedDomainName );
+	};
+
+	const renderAdvancedTransferOptions = () => {
+		// translators: domain transfer lock
+		const disabledLockLabel = __( 'Transfer lock off' );
+		// translators: domain transfer lock
+		const enabledLockLabel = __( 'Transfer lock on' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Transfer lock: on' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/advanced-transfer-options-component)